types/google-maps-web/index.d.ts

```diff
@@ -0,0 +1,7699 @@
+// Type definitions for non-npm package Google Maps JavaScript API 3.42
+// Project: https://developers.google.com/maps/
+// Definitions by: Justin Poehnelt <https://github.com/jpoehnelt>
+//                 Alex Muramoto <https://github.com/amuramoto>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Generated by an automated process. DO NOT EDIT!
+
+/**
+ * @license
```

Comment:
DT index files typically don't include a licence because the publisher generates one that uses MIT, for example: https://unpkg.com/browse/@types/aframe@1.0.2/LICENSE

Is that OK?